web/src/styles/components/profile-tab.scss

```diff
@@ -0,0 +1,3 @@
+.align-text-left {
```

Comment:
This class already exists, do a global search for `text-align: left` in the project

Reply:
it's `.tal` in `flex.scss`